webpages/settings/style.css

```
@@ -316,13 +316,16 @@ body.iframe .search-box input {
 }
 
 .permissions div.main {
-  justify-content: center;
-  display: grid;
-  grid-template-rows: 14rem 10rem;
+  display: block;
+  box-sizing: border-box;
+  max-width: fit-content;
+  margin: auto;
+  padding: 1rem;
```

```suggestion
  padding: 2rem;
```
(same as Licenses page)